aspnetcore/fundamentals/environments.md

```diff
@@ -175,8 +175,8 @@ The [About page](https://github.com/dotnet/AspNetCore.Docs/tree/main/aspnetcore/
 To set the environment in an [Azure App Service](https://azure.microsoft.com/services/app-service/) app by using the portal:
 
 1. Select the app from the **App Services** page.
-1. In the **Settings** group, select **Configuration**.
-1. In the **Application settings** tab, select **New application setting**.
+1. In the **Settings** group, select **Environment variables**.
+1. In the **App settings** tab, select **+ Add**.
```

Comment:
Replace most of that with 
see [Configure app settings](https://learn.microsoft.com/en-us/azure/app-service/configure-common?tabs=portal#configure-app-settings)

That Azure page is maintained. The UI changes every couple of months. No need to spoon feed them.

Reply:
Good idea, I'll do that next time.